src/hyperfine/app.rs

```diff
@@ -115,6 +115,16 @@ fn build_app() -> App<'static, 'static> {
                 .short("i")
                 .help("Ignore non-zero exit codes."),
         )
+        .arg(
+            Arg::with_name("time-unit")
+                .long("time-unit")
+                .short("u")
+                .takes_value(true)
+                .value_name("millisecond|second")
+                .possible_values(&["millisecond", "second"])
+                .help( "Set the time unit used for CLI output and Markdown export.",
```

Comment:
I'd remove the part about "CLI output" and "Markdown output". Maybe just: "Explicitly set the unit of time. Possible values: .."?